tests/acceptance/content.feature

```diff
@@ -0,0 +1,38 @@
+Feature: content
+  In order to manage content article in the web
+  As an owner
+  I need to create modify trash publish and Unpublish content article
+
+  Background:
+    Given Joomla CMS is installed
+    When Login into Joomla administrator with username "admin" and password "admin"
+    Then I see administrator dashboard
+
+  Scenario: Create an Article
+    Given There is a Add Content link
+    When I add the content with title "My_Article"
+    And I add content body "This is my first article"
+    And I save an article
+    Then I should see the "Article successfully saved." message
+
+  Scenario: Feature an Article
+    Given I search and select content article with title "My_Article"
+    When I feature the content with title "My_Article"
+    Then I save and see the "1 article featured." message
+
+  Scenario: Modify an article
+    Given I select the content article with title "My_Article"
+    And I set access level as a register
```

Comment:
Would it have sense to have the access level as a variable? 

`And I set access level as a "register"`

```php
/**
 * @Given I set access level as a register
 */
public function iSetAccessLevelAsA($accessLevel)
...
```